crab/submit.py

```diff
@@ -113,6 +126,16 @@ def verifySample(sample):
                 continue
 
             requestName = sample.split('/')[1]
+            if args.data:
+                requestName = sample.split('-')[0].replace('/', '_')[1:]
+
+            # LHE container not available for diboson samples
+            addSignalLHE = 'addSignalLHE=1'
```

Comment:
better to use 'True' and 'False' instead of 0 and 1 since it is a boolean?

Reply:
No, it works with integers.
I am not sure the python2 parser can interpret the string `True`.